step-by-step/04-check-compat.md

```diff
@@ -13,8 +13,6 @@ We are going to check if your device is supported.
 3. Check that `Arch` says `arm64-v8a`
 4. If it does not say `arm64-v8a` you cannot patch YouTube on your device. Please patch it on another device or your PC.
 
-==- What does `arch` mean? Why is `armeabi-v7a` not supported?
-`arch` is the architecture of your CPU. In layman terms, it is "the language of the CPU", if you don't know that language, you cannot communicate with that CPU. 
-
-The ReVanced team will move to `arsclib` soon™, which supports `armeabi-v7a`. For now, you can patch on a computer with `revanced-cli` or `revanced-builder` instead.
+==- What does `arch` mean? Why is `arm64-v8a` required?
+`arch` is the architecture of your CPU. In layman terms, it is "the language of the CPU". If you don't know that language, you cannot communicate with that CPU. If your device does not support `arm64-v8a` you can patch on a computer with the [`revanced-cli`](https://sodawithoutsparkles.github.io/revanced-troubleshooting-guide/06-revanced-cli/), [`Taku's auto-cli`](https://github.com/taku-nm/auto-cli) or [`revanced-builder`](https://github.com/reisxd/revanced-builder).
```

Comment:
Is the revanced-cli link correct? I wasn't sure if it should link to the `.md` file.

Reply:
Its fine, but using `/06-revanced-cli.md` would be better because it retains compatibility with github's view